content/en/mpas/getting_started.md

```diff
@@ -0,0 +1,394 @@
+      message: Initializing Project repository
+      name: mpas-admin
+  prune: true
+EOF
+```
+
+Then, apply the project to the clustern in a gitOps fashion:
+
+```bash
+git add --all && git commit -m "Add podinfo project" && git push
+```
+
+`Flux` will detect the changes and apply the project to the cluster.
+
+This will create in the cluster a `namespace` for the project, a `service account`, and RBAC.
+It will also create a GitHub repository for the project, and configure `Flux` to manage the project's resources.
+
+3. Add the needed secrets to the namespace
+
+`Flux` is used to deploy all workloads in a gitOps way. In order for flux to access the internal
+registry, we have to provide the certificate to use `https`.
+
+```bash
+kubectl get secret ocm-registry-tls-certs --namespace=mpas-system -o yaml | sed 's/namespace: 
+.*/namespace: mpas-podinfo-application/' | kubectl apply -f -
+```
+
+We also need a secret in the project namespace that will be used to communicate with github:
+
+```bash
+kubectl create secret generic \
+  github-access \
+  --from-literal=username=$GITHUB_USER \
+  --from-literal=password=$GITHUB_TOKEN \
+  -n mpas-podinfo-application
+```
+
+**Note** The credentials shall have access to github packages.
+
+As part of step 2, a `service account` was created for the project. We will use this service account
+to provide the necessary permissions to pull from the `ghcr` registry.
+
+First create a secret containing the credentials for the service account:
+
+```bash
+kubectl create secret docker-registry github-registry-key --docker-server=ghcr.io \
+  --docker-username=$GITHUB_USER --docker-password=$GITHUB_TOKEN \
+  --docker-email=<MY_EMAIL> -n mpas-podinfo-application
+```
+
+Then, patch the service account to use the secret:
+
+```bash
+kubectl patch serviceaccount mpas-podinfo-application -p '{"imagePullSecrets": [{"name": "github-registry-key"}]}' \
+  -n mpas-podinfo-application
+```
+
+4. Clone the project repository
+
+```bash
+git clone https://github.com/$GITHUB_USER/mpas-podinfo-application
+cd mpas-podinfo-application
+```
+
+5. Add the podinfo component subscription
+
+Create a file under `./subscriptions/` that will contains the subscription declaration.
+
+```bash
+cat <<EOF >> ./subscriptions/podinfo.yaml
+apiVersion: delivery.ocm.software/v1alpha1
+kind: ComponentSubscription
+metadata:
+  name: podinfo-subscription
+  namespace: mpas-podinfo-application
+spec:
+  interval: 30s
+  component: mpas.ocm.software/podinfo
+  semver: ">=v1.0.0"
+  source:
+    url: ghcr.io/open-component-model/mpas
+    secretRef:
+      name: github-access
+  destination:
+    url: ghcr.io/$GITHUB_USER
+    secretRef:
+      name: github-access
+EOF
+```
+
+Then, apply the `ComponentSubscription` to the project in a gitOps fashion:
+
+```bash
+git add --all && git commit -m "Add podinfo subscription" && git push
+```
+
+`Flux` will detect the changes and apply the subscription to the cluster.
```

Comment:
Perhaps add a way to "kick" flux into action so people don't have to wait?

```
flux reconcile source git flux-system
```

Reply:
Yeah, but that means stating `flux cli` is required. It will make things easier but it's not mandatory. I did not want to have a long list of requirement.